src/Core/Framework/Store/Api/ExtensionStoreActionsController.php

```diff
@@ -130,6 +130,8 @@ public function uninstallExtension(string $type, string $technicalName, Request
             $context
         );
 
+        $this->cache->delete(StoreClient::EXTENSION_LIST_CACHE);
```

Comment:
I would subscribe to DAL written event to plugin/app entity and then clear the cache there. it's pretty uggly here and cli commands are not considered

Reply:
ok let me have a look at that approach.